src/targets/gpu/fuse_mlir.cpp

```diff
@@ -68,7 +92,7 @@ namespace {
 
 MIGRAPHX_PRED_MATCHER(is_mlir_conv, instruction_ref ins)
 {
-    if(ins->name() != "convolution")
+    if(ins->name() != "convolution" && ins->name() != "quant_convolution")
```

Comment:
Use `and` instead of `&&`.

Reply:
Sorry, has not been very careful with this convention. I'll pay more attention to this.